apps/backend/src/reconciliation/reconciliation.service.ts

```diff
@@ -0,0 +1,84 @@
+import { POSDepositEntity } from './entities/pos-deposit.entity';
+import { TransactionEntity } from './entities/transaction.entity';
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { CashDepositEntity } from './entities/cash-deposits';
+
+@Injectable()
+export class ReconciliationService {
+  constructor(
+    @InjectRepository(TransactionEntity)
+    private salesRepo: Repository<TransactionEntity>,
+    @InjectRepository(POSDepositEntity)
+    private TDI34: Repository<POSDepositEntity>,
+    @InjectRepository(CashDepositEntity)
+    private TDI17: Repository<CashDepositEntity>
+  ) {}
+
+  //TODO ADD DTO AND RETURN TYPE
+  async addSales(body: any[]) {
```

Comment:
we wont need this for sales, it should pipe in through the API

Reply:
if we are transporting data dumps, then it should live else where and not with production code.